nvm.sh

```diff
@@ -1863,8 +1872,7 @@ BEGIN {
       fmt_version = fmt_installed;
     }
 
-    padding = (!has_colors && is_installed) ? "" : "  ";
-
+    padding = (is_installed && !has_colors) ? "" : "  ";
```

Comment:
nbd, but why flip these conditions?

Reply:
Not really flipping, just swapping the LHS and RHS of the `&&` operator. This slightly improves the clarity and makes it clear that only `!has_colors` is the negating part. This is something I should have done in the first place.